src/app/mainview/mainview.component.ts

```diff
@@ -48,6 +49,10 @@ export class MainviewComponent implements OnInit, OnDestroy {
     @ViewChild('CitationForm') citationForm;
     public newCitForm: FormGroup;
     public title: string;
+    public statisticGroup: any;
+    public item: any;
+    public regRegion: any;
+    public clone: boolean;
```

Comment:
Same as above, I don't think these variables are actually used anywhere else in the mainview component.